src/components/FilterButtons.vue

```diff
@@ -34,7 +39,16 @@ export default {
     secondButtonTitle: String,
     firstOption: String,
     secondOption: String,
-    options: String
+    options: String,
+    checked: String
+  },
+  methods: {
+    changeRadioInput(inputValue) {
+      this.$emit("onChangeInput", inputValue);
```

Comment:
Following the same logic as for routes and actions, consider creating an "events" file under "src/core" and keep the event names there.
Consult https://vuejs.org/v2/api/#v-on
"Shorthand Dynamic Event"

Reply:
Fixed